src/main/java/bio/terra/janitor/db/JanitorDao.java

```diff
@@ -139,31 +169,37 @@ public void createResource(TrackedResource resource, Map<String, String> labels)
   }
 
   /** Retrieve the {@link CleanupFlight}s associated with the tracked resource. */
-  public List<CleanupFlight> getFlights(TrackedResourceId trackedResourceId) {
+  @Transactional(propagation = Propagation.SUPPORTS)
+  public List<CleanupFlight> retrieveFlights(TrackedResourceId trackedResourceId) {
     String sql =
         "SELECT flight_id, flight_state from cleanup_flight WHERE tracked_resource_id = :tracked_resource_id;";
     MapSqlParameterSource params =
         new MapSqlParameterSource().addValue("tracked_resource_id", trackedResourceId.uuid());
     return jdbcTemplate.query(sql, params, CLEANUP_FLIGHT_ROW_MAPPER);
   }
 
-  public Optional<CleanupFlightState> getFlightState(String flightId) {
+  @Transactional(propagation = Propagation.SUPPORTS)
+  public Optional<CleanupFlightState> retrieveFlightState(String flightId) {
```

Comment:
just curious: Why we want to rename get to retrive for all of them?

Reply:
I was thinking that `get` might imply it's like a getter where the info is in memory, whereas `retrieve` sounds more like a network operation. And I think I was already being inconsistent, so moved to all `retrieve`. sg?